src/components/molecules/InternalChevronAccordion/InternalChevronAccordion.tsx

```diff
@@ -55,18 +55,14 @@ export const StyledAccordionButton = styled(
   }
 `;
 
-export const StyledAccordion = styled(InternalAccordion)<
-  OakBoxProps & FlexStyleProps
->`
+const StyledAccordion = styled(InternalAccordion)<FlexStyleProps>`
   ${StyledAccordionUnderline} {
     visibility: hidden;
   }
   ${StyledAccordionButton}:focus-visible ~ ${StyledAccordionUnderline} {
     visibility: visible;
   }
-  ${StyledAccordionButton}:active ~ ${StyledAccordionUnderline} {
-    visibility: visible;
```

Comment:
why have you removed this? the figma states the underline should be there when the accordion is clicked